src/randoop/main/GenInputsAbstract.java

```diff
@@ -326,13 +289,6 @@ public GenInputsAbstract(String command, String pitch,
   @Option("Whether to output error-revealing tests")
   public static boolean no_error_revealing_tests = false;
```

Comment:
Currently this appears in the option group "outputting the JUnit tests", which also has details like the file name for the test files.  I think that conceptual issues like which tests to output and how to treat exceptions should be separated from filename handling and implementation details like whether JUnit is used.  I suggest that the former should be made into a new option group that appears earlier, probably right before or right after the "limiting test generation"  option group.